apps/heureka/src/components/issueMatches/IssueMatchesListItem.jsx

```diff
@@ -5,10 +5,17 @@
 
 import React from "react"
 import { DataGridRow, DataGridCell } from "@cloudoperators/juno-ui-components"
-import { listOfCommaSeparatedObjs, formatDate } from "../shared/Helper"
+import { listOfCommaSeparatedObjs, formatDate, cellSeverityClasses } from "../shared/Helper"
 import constants from "../shared/constants"
 import { useGlobalsActions, useGlobalsShowPanel, useGlobalsShowIssueDetail } from "../../hooks/useAppStore"
 
+// Helper function to extract the last four parts of a URL - used for displaying ccrn
+export const extractLastFourParts = (url) => {
```

Comment:
The name of this helper is maybe a bit too literal. What if you want to change it so it extracts only the last three tokens? My suggestion would be to choose a more semantic name that describes what you do but not how you do it.
```suggestion
export const extractResourceInfo = (url) => {
```